aicsimageio/readers/reader.py

```diff
@@ -132,8 +159,16 @@ def dask_data(self) -> da.core.Array:
 
     @property
     def data(self) -> np.ndarray:
+        global USE_DASK
```

Comment:
I may be wrong but I don't think you need this line - you should only need it if you intend to modify the variable.

Reply:
Looks like you are correct. Removed, tested, and repushing now.